### PR TITLE
Snips to Filter Duplicate Values from Array

### LINKED ISSUE #15: Add snip to filter duplicate values from an array

### PR DESCRIPTION
**Pull request by SarojWasti**
Close #15

### CODE REVIEW

**Comment by SarojWasti** [comment]
@DevTMK I have opened PR for issue 15. You can review it.

File: src/filterDuplicate.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Filter Duplicate Values
+Filters Duplicate Values From An Array.
+
+## Result
+```js
+  filterDups[1,2,2,3,4]; // '[1,2,3,4]'
```
Comment (DevTMK) [inline src/filterDuplicate.md:6]: Invalid code, should be a proper function call

💭 Always write readable variable names

File: src/filterDuplicate.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+```
+
+## Snip Langauge
+* [JavaScript](#javascript)
+* [Python](#python)
+
+### JavaScript
+```js
+  function removeDuplicate(arr){
+    var uniqArr = [...new Set(arr)];
```
Comment (DevTMK) [inline src/filterDuplicate.md:16]: Not getting desired output, please check your code once

**Review by DevTMK — CHANGES_REQUESTED** [review]

**Review by DevTMK — APPROVED** [review]
Thanks @SarojWasti, Merging

**Comment by SarojWasti** [comment]
@DevTMK Thanks for merging.

**Pull request merged**